tests/test_database.py

```diff
@@ -163,6 +164,28 @@ def test_drop_and_pick_tables():
     assert 'segments' not in db
 
 
+def test_file_duration():
+
+    db = pytest.DB
+
+    # duration of single file
+
+    file = db.files[0]
+    full_file = os.path.join(db.root, file)
+    dur = pd.to_timedelta(audiofile.duration(full_file), unit='s')
+    assert db.file_duration(file) == dur
+    assert db.file_duration(full_file) == dur
```

Comment:
I think you should repeat the last two lines in order to check that a second call still returns the same duration.

Reply:
done